File: PrimeNumber.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var isPrime bool
	count := 2
	fmt.Printf("1\t2\n")
	for i := uint64(3); i < math.MaxUint64; i += 2 {
		isPrime = true
		for j := uint64(3); j*j <= i; j += 2 {
			if i%j == 0 {
				isPrime = false
				break
			}
		}
		if isPrime {
			fmt.Printf("%v\t%v\n", count, i)
			count++
		}
	}
}

File: PrimeNumber.c
#include <stdio.h>
#include <math.h>

int main() {
    unsigned n;
    int count = 2;
    scanf("%d", &n);
    printf("1\t : 2\n");
    for (unsigned i = 3; i <= n; i += 2) {
        int isPrime = 1;
        if (i % 2 == 0) {
            isPrime = 0;
        } else {
            for (unsigned j = 3; j*j <= i; j += 2) {
                if (i % j == 0) {
                    isPrime = 0;
                    break;
                }
            }
        }
        if (isPrime == 1) {
            printf("%d\t : %d\n", count, i);
            count++;
        }
    }
    return 0;
}
